react-cors-spa/src/App.js

```diff
@@ -16,31 +16,36 @@ COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER I
 TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 */
 
-import logo from './logo.svg';
-import logoS3 from './logoS3.png';
-import logoCF from './logoCloudFront.png';
-import './App.css';
-import axios from 'axios';
-import { useState, useEffect } from 'react';
+import axios from "axios";
+import { Quotes, Doges } from "./ApiDisplay";
```

Comment:
Casing doesn't match actual file name

Reply:
And wasn't letting me build the app

Reply:
I had to rename the file to ApiDisplay.js, and push this change to git via my personal computer and work laptop

Does git not track file name changes?

Reply:
It should. I just renamed a bunch of files the other day and committed the changes.